docs/prerequisites.rst

```diff
@@ -0,0 +1,77 @@
+Prerequisites
+===============
+
+- Tezos
+
+Mac:
+Brew:
+
+```
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+````
+
+External libs:
+```
+brew install hidapi libev wget
+```
+
+Install tezos:
+```
+git clone https://gitlab.com/tezos/tezos.git
+cd tezos
+git checkout latest-release
+```
+
+Install Ocaml:
+```
+brew install ocaml
+wget https://github.com/ocaml/opam/releases/download/2.0.7/opam-2.0.7-x86_64-macOS
+sudo mv opam-2.0.7-x86_64-macOS /usr/local/bin/opam
+cd /usr/local/bin/
+sudo chmod a+x opam
+opam init
+opam update
+opam update
+eval $(opam env)
+opam switch create 4.09.1
+eval $(opam env)
+```
+
+install Rust:
+```
+sudo wget https://sh.rustup.rs/rustup-init.sh
+sudo chmod +x rustup-init.sh
+./rustup-init.sh --profile minimal --default-toolchain 1.52.1 -y
```

Comment:
./rustup-init.sh --profile minimal --default-toolchain 1.60.0 -y

Apart from the hints with missing deps on mac shouldn't we rather point to the official docs?